src/test/javascript/spec/component/course/course-management-exercises.component.spec.ts

```diff
@@ -82,4 +84,16 @@ describe('Course Management Exercises Component', () => {
         expect(comp.courseId).to.equal(course.id);
         expect(findStub).to.have.been.calledWith(course.id);
     });
+
+    it('should open search bar on button click', () => {
+        fixture.detectChanges();
+        const button = fixture.debugElement.nativeElement.querySelector('#toggleSearchButton');
+        button.click();
+        fixture.detectChanges();
+
+        const searchBar = fixture.debugElement.nativeElement.querySelector('jhi-course-management-exercises-search');
+
+        expect(comp.showSearch).to.equal(true);
+        expect(searchBar).to.not.equal(null);
```

Comment:
```suggestion
        expect(searchBar).not.toBe(null);
```

Reply:
Since this test was still using sinon/chai this was not possible.
So I converted them to Jest and also did this.